Makefile

```diff
@@ -85,10 +85,21 @@ acceptance: ## Run acceptance tests
 
 # Add @focus above the feature you're hacking on to use this
 .PHONY: focus-acceptance
-focus-acceptance: ## Run acceptance tests with @focus tag
```

Comment:
Intentional removal?

Reply:
Yeah. The reasoning is that a dev would use one of the new targets and have little need to run this one directly any more.

Reply:
Hmm ok. It seems odd to remove the docs for it instead of just removing the command.

Reply:
I'll put the docs back since maybe it's still useful by itself.